models/webhooks/store_option_updated.yml

```diff
@@ -0,0 +1,33 @@
+type: object
+properties:
+  store/option/updated:
+    description: Fires when product options overrides are edited. `attributes` and `context` fields are present for only override updates. 
+    type: object
+    properties:
+      scope:
+        type: string
+      store_id:
+        type: string
+      data:
+        type: object
+        properties:
+          type:
+            type: string
+          id:
+            type: integer
+          affected_product_ids: 
```

Comment:
Should this be `product_ids` ?

Reply:
We have `affected_product_ids` in implementation. @kristinapototska are we ok with that name or should we change to `product_ids` in our implementation?